MyApplication/app/src/main/java/test/test/com/myapplication/widgets/MyApplication.java

```diff
@@ -0,0 +1,23 @@
+package test.test.com.myapplication.widgets;
+
+import android.app.Application;
+import android.content.Context;
+
+/**
+ * Created by ShaharAlush on 29/01/2017.
+ */
+
+public class MyApplication extends Application {
```

Comment:
When moving to multidex in gradle you need your application class to extend multidexapplication...